### DISCUSSION TITLE
打包单个文件太大，导致失败

### DISCUSSION THREAD

**Discussion by qiliangya** [umijs/umi#7378] [Q&A]
``` /.umi-production/core/routes.ts as it exceeds the max of 500KB ```

路由文件单个超出了500k ，有什么办法解决吗

**Comment by xinre** [comment]
请问解决了吗？ 我升级到umi4碰到这个问题了。  试了修改 .env  BABELRC=true
 还有创建  .babelrc/babel.config.js 文件 加   "compact": false。 都没用

**Comment by fz6m** [comment]
如果有 babel 压缩大小问题，一般只是警告，可以忽略，如果会报错退出，需要给一个最小复现，否则请自行拆包。

或者尝试使用其他的转译器：[`srcTranspiler`](https://umijs.org/docs/api/config#srctranspiler)
